prometheus_flask_exporter/__init__.py

```diff
@@ -353,21 +354,28 @@ def export_defaults(self, buckets=None, group_by='path',
 
         labels = self._get_combined_labels(None)
 
-        histogram = Histogram(
+        request_duration_metric = Histogram(
             '%shttp_request_duration_seconds' % prefix,
             'Flask HTTP request duration in seconds',
             ('method', duration_group_name, 'status') + labels.keys(),
             registry=self.registry,
             **buckets_as_kwargs
         )
 
-        counter = Counter(
+        request_total_metric = Counter(
             '%shttp_request_total' % prefix,
             'Total number of HTTP requests',
             ('method', 'status') + labels.keys(),
             registry=self.registry
         )
 
+        request_exceptions_metric = Counter(
+            '%shttp_request_exceptions_total' % prefix,
+            'Total number of HTTP requests',
+            ('method', 'path', 'status') + labels.keys(),
```

Comment:
Can we please leave the path off for now to be consistent with the other default metrics?
Perhaps another change could introduce a new parameter to also include a path (or endpoint) in the default labels, but I wouldn't add it here unconditionally.
I remember there were requests from people who have a large range of paths handled by a relatively small number of endpoints.